gnocchi/tests/functional/gabbits/archive.yaml

```diff
@@ -561,8 +561,8 @@ tests:
       status: 201
       response_json_paths:
           $.definition[0].points: 1000
-          $.definition[0].granularity: "0:00:04"
-          $.definition[0].timespan: "1:06:40"
+          $.definition[0].granularity: "0:00:03.600000"
+          $.definition[0].timespan: "1:00:00"
```

Comment:
Yes. I read that twice but we always supported nanoseconds aggregation so if you request 1 hours and 1k points this is what will happen. I guess it was not clear when we wrote the test that nanoseconds aggregation was expected to work and no rounding was done.

The underlying aggregation done was already 3.6s so that does not change. We just fix the returned result on this API call.